manifests/a/angeldev0/messyorganizer/1.0.0/angeldev0.messyorganizer.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: angeldev0.messyorganizer
+PackageVersion: 1.0.0
+InstallerType: inno
+Installers:
+- InstallerUrl: https://github.com/4ngel2769/messy-file-organizer/releases/latest/download/MessyFileOrganizer-setup-x64.exe

```

```suggestion
- InstallerUrl: https://github.com/4ngel2769/messy-file-organizer/releases/download/v1.0.0-beta.1/MessyFileOrganizer-setup-x64.exe
```
Using a versioned URL is preferred to using a `latest` URL, to ensure the package doesn't encounter hash mismatch errors when a new version is released